frontend/src/lib/components/ChildrenRegistration.svelte

```diff
@@ -450,7 +450,7 @@ async function submitData(): Promise<void> {
 								.additional_answer}
 							label={element?.text?.[i18n.locale].question}
 							textTrigger={element.additional_option}
-							required={element.component === 'fileupload' ? false : true}
+							required={(element.component === 'fileupload' || element.required === false) ? false : true}
```

Comment:
it looks like the fileupload component created above already has `.required=false`, so I think this can now be simplified (I guess this logic predates a required field?):
```suggestion
							required={element.required}
```

Reply:
Agreed, it makes more sense this way. The actions of interacting with the interface and what it does will align.